src/classes/STG_InstallScript.cls

```diff
@@ -268,4 +270,18 @@ global without sharing class STG_InstallScript implements InstallHandler {
             upsert orgSettings;
         }
     }
+    
+    /*******************************************************************************************************
+    * @description Populate Default Contact Language Fluency custom setting when upgrading to 1.74
+    * @return void
+    */
+    private static void setDefaultContactLanguageFluency(Version previousVersion) {
+        if(previousVersion != null && previousVersion.compareTo(new Version(1,74)) < 0) {
```

Comment:
This will definitely need to change if this PR doesn't make it in 1.74!!!

Reply:
I'm not seeing how we are gonna make it into 1.74 at this point. Time is running out.

Reply:
Yep, I'll change it.

Reply:
Done